### PR TITLE
[Model] Create Group Model

### PR DESCRIPTION
**Pull request by raphaelsattler**
Created Group model with validations and its tests.

### CODE REVIEW

File: spec/factories.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
+  factory :group do
+    name { "Sellers" }
```
Comment (kaiomagalhaes) [inline spec/factories.rb]: you should user Faker here as well

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: spec/factories.rb (hunk 1 above)
Comment (Heirian) [reply spec/factories.rb]: Why this conversation was marked as resolved without a faker name?

File: db/migrate/20180920153238_create_group.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateGroup < ActiveRecord::Migration[5.2]
+  def change
+    create_table :groups do |t|
+      t.string :name
+      t.boolean :active, default: true
+
```
Comment (Heirian) [inline db/migrate/20180920153238_create_group.rb]: Where is the foreign key of a parent group?

File: spec/models/group_spec.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe Group, type: :model do
+  it { should validate_presence_of(:name) }
```
Comment (RodrigoNM) [inline spec/models/group_spec.rb]: According to http://www.betterspecs.org On one line expectations or with implicit subject we should use is_expected.to

File: spec/models/group_spec.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+  it { should validate_uniqueness_of(:name) }
+
+  it "the active should be true by default" do
+    group = create(:group)
```
Comment (RodrigoNM) [inline spec/models/group_spec.rb]: It's better to use let to initialize actions

**Review by kaiomagalhaes — APPROVED** [review]

**Review by RodrigoNM — APPROVED** [review]

**Review by Heirian — APPROVED** [review]

**Pull request merged**